demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -15,64 +15,56 @@ class ReadMeValidator:
             export DEMISTO_README_VALIDATION=True
     """
 
-    def __init__(self, file_path):
-        self.file_path = file_path
+    def __init__(self, file_path: str):
+        self.content_path = get_content_path()
+        self.file_path = Path(file_path)
+        self.pack_path = self.file_path.parent
 
-    def is_valid_file(self):
+    def is_valid_file(self) -> bool:
         """Check whether the readme file is valid or not
+        Returns:
+            bool: True if env configured else Fale.
         """
-        if os.environ.get('DEMISTO_README_VALIDATION'):
-            is_readme_valid = all([
-                self.is_mdx_file(),
-            ])
-            return is_readme_valid
+        if os.environ.get('DEMISTO_README_VALIDATION') or os.environ.get('CI'):
+            return self.is_mdx_file()
         else:
             return True
 
     def is_mdx_file(self) -> bool:
-        ready, is_valid = self.are_modules_installed_for_verify()
-        if ready:
-            mdx_parse = f'{os.path.dirname(os.path.abspath(__file__))}/../mdx-parse.js'
+        valid = self.are_modules_installed_for_verify()
+        if valid:
+            mdx_parse = Path(__file__).parent.parent / 'mdx-parse.js'
             # run the java script mdx parse validator
-            res = subprocess.run(['node', mdx_parse, '-f', self.file_path], text=True, timeout=10,
-                                 capture_output=True)
-            if res.returncode != 0:
-                print_error(f'Failed verifying README.md, Path: {self.file_path}. Error Message is: {res.stderr}')
-                is_valid = False
-        return is_valid
+            _, stderr, is_valid = run_command_os(f'node {mdx_parse} -f {self.file_path}', cwd=self.content_path)
```

Comment:
You need to set the environment variable when running this os command. Otherwise node will not find the node_modules in the content_path dir:
`NODE_PATH=<content_path>/node_modules:$NODE_PATH`